ci/taos/plugins-good/pr-format-doxygen-tag.sh

```diff
@@ -53,6 +53,11 @@ function pr-format-doxygen-tag(){
                     # Append a doxgen rule step by step
                     doxygen_basic_rules="@file @brief" # @file and @brief to inspect file
                     doxygen_advanced_rules="@author @bug" # @author, @bug to inspect file, @brief for to inspect function
+
+                    # When check_result is skip, set value to success(default)
```

Comment:
Please update the statement. 


`# When a current file is a source code, change a default value of $check_result from 'skip' to 'success'.`